packages/ui/src/modal/acceptOffer/Progress.tsx

```diff
@@ -0,0 +1,65 @@
+import { Anchor, Flex, Text } from '../../primitives'
+import React, { FC } from 'react'
+import { AcceptOfferStep } from './AcceptOfferModalRenderer'
+// @ts-ignore
+import confirmingProgress from 'url:../../../assets/confirmingProgress.gif'
+// @ts-ignore
+import finalizingProgress from 'url:../../../assets/finalizingProgress.gif'
+
+type Props = {
+  acceptOfferStep: AcceptOfferStep
+  etherscanBaseUrl?: string
+}
+
+export const Progress: FC<Props> = ({ acceptOfferStep, etherscanBaseUrl }) => {
+  return (
+    <Flex
+      direction="column"
+      css={{
+        alignItems: 'center',
+        gap: '$4',
+        mt: '$5',
+        mb: '$3',
+      }}
+    >
+      {acceptOfferStep == AcceptOfferStep.ApproveMarketplace && (
+        <>
+          <Text style="h6">Approve X2Y2 to access item in your wallet</Text>
+          {/* <img style={{ height: 100 }} src={confirmingProgress} /> */}
```

Comment:
Seems like we can probably remove this commented out copy. We should also not hardcode X2Y2 here.